frontend/pages/viewer-registration/viewer-registration.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FormEvent, useState } from 'react'
+import styles from './viewer-registration.module.css'
+import PasswordAid from './password-aid/password-aid'
+import { EnrollmentCTA, EnrollmentHeader, SubmitButton, TextInput } from '../../shared-components'
+import { CTATypes, InputNames, TooltipTypes } from '../../models'
+
+export default function ViewerRegistration() {  
+  const { FIRST_NAME, LAST_NAME, EMAIL_ADDRESS, PHONE_NUMBER, CREATE_PASSWORD, CONFIRM_PASSWORD } = InputNames
+  const { registration, inputLine } = styles
+  const passwordAidId: string = 'passwordAid'
+  
+  const [isSubmitted, setSubmission] = useState(false)
+  const [isPasswordShown, setPasswordDisplay] = useState(false)
+
+  function handlePasswordDisplay(): void {
+    setPasswordDisplay(!isPasswordShown)
+  }
+
+  function handleSubmit($event: FormEvent<HTMLFormElement>): void {
+    // TODO: handle form submission - set input values to state onChange
+    $event.preventDefault()
+    setSubmission(true)
+  }
+
+  return (
+    <section className={registration}>
+      <EnrollmentHeader
+        headerText="Register: Viewer Account" 
+        tooltip={TooltipTypes.VIEWER}
+      />
+      <form onSubmit={handleSubmit}>
```

Comment:
please remove the form element just use a button with onClick.

Reply:
will then reduce the fieldset use and simplify the component

Reply:
Happy to use the button! but why no fieldset? I need a container of alike inputs & it's better to use a semantic element than a div or something

Reply:
Oh an as for the form element it's important for accessibility that things are semantically what they are performing as - if this is not a form element it would be inaccessible to screen reader users & fail any audit (WCAG 1.3.1)